index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    
+    <h1>Hello World, we are software developers!</h1>
+
+      <header id="header"> Handcrafted, home-made masterpieces
+        <nav id="nav-bar">
+          <ul>
+            <li class="nav-link"><a href="#">Home</a></li>
+            <li class="nav-link"><a href="#news">News</a></li>
+            <li class="nav-link"><a href="#contact">Contact</a></li>
+            <li class="nav-link"><a href="#about">About</a></li>
+          </ul>
+        </nav>
+        <img src="logo-1340524__340.webp" id="header-img" alt="">
```

Comment:
Please place this image in the sections tag if its not a logo.

Reply:
- [ ] You have done a great job to be polite when requesting for the changes 👏 . However, the student may not be able to understand why this should be done. Therefore i suggest that you provide a detailed explaination to why putting the image in the sections is important 👍 .